### PR TITLE
fix: remove cookie

### PR DESCRIPTION
**Pull request by mubbsharanwar**
-remove marketingEmailsOptIn cookie on successful registration
- fix tests

#### How Has This Been Tested?

Test through unit tests and manual testing

### CODE REVIEW

**Review by attiyaIshaque — APPROVED** [review]

**Review by syedsajjadkazmii — CHANGES_REQUESTED** [review]
please resolve this

File: src/common-components/RedirectLogistration.jsx [hunk 1]
```diff
@@ -34,6 +34,9 @@ const RedirectLogistration = (props) => {
       finalRedirectUrl = redirectUrl;
     }
 
+    // remove marketingEmailsOptIn cookie, setting on SSO registration
+    removeCookie('marketingEmailsOptIn');
```
Comment (syedsajjadkazmii) [inline src/common-components/RedirectLogistration.jsx:38]: please add this in registration form success useEffect

**Review by syedsajjadkazmii — CHANGES_REQUESTED** [review]

File: src/register/RegistrationPage.jsx [hunk 2]
```diff
@@ -187,6 +187,9 @@ const RegistrationPage = (props) => {
 
       // This is used by the "User Retention Rate Event" on GTM
       setCookie(getConfig().USER_RETENTION_COOKIE_NAME, true);
+
+    // remove marketingEmailsOptIn cookie, setting on SSO registration
+    removeCookie('marketingEmailsOptIn');
```
Comment (syedsajjadkazmii) [inline src/register/RegistrationPage.jsx:192]: linting needs to be resolved here

File: src/register/RegistrationPage.jsx [hunk 3]
```diff
@@ -187,6 +187,9 @@ const RegistrationPage = (props) => {
 
       // This is used by the "User Retention Rate Event" on GTM
       setCookie(getConfig().USER_RETENTION_COOKIE_NAME, true);
+
+    // remove marketingEmailsOptIn cookie, setting on SSO registration
```
Comment (syedsajjadkazmii) [inline src/register/RegistrationPage.jsx:191]:
```suggestion
    // remove marketingEmailsOptIn cookie that was set on SSO registration flow
```

**Review by syedsajjadkazmii — APPROVED** [review]

**Pull request merged**